nvme-print-stdout.c

```diff
@@ -2167,7 +2167,9 @@ static void stdout_id_ctrl_cqes(__u8 cqes)
 static void stdout_id_ctrl_oncs(__le16 ctrl_oncs)
 {
 	__u16 oncs = le16_to_cpu(ctrl_oncs);
-	__u16 rsvd = (oncs & 0xFE00) >> 9;
+	__u16 rsvd = (oncs & 0xF800) >> 11;
+	__u16 afc  = (oncs & 0x400) >> 10;
+	__u16 csa  = (oncs & 0x200) >> 9;
```

Comment:
I see these are following the existing style, but seems like this would be simpler?
```c
__u16 rsvd = oncs >> 11;
bool afc = oncs >> 10 & 1;
bool csa = oncs >> 9 & 1;
```
But probably would be even better to use the constants you defined in `libnvme`, e.g. `NVME_CTRL_ONCS_COPY_SINGLE_ATOMICITY`.